components/movementsensor/gpsnmea/serial.go

```diff
@@ -34,10 +35,11 @@ type SerialNMEAMovementSensor struct {
 	data                    GPSData
 	activeBackgroundWorkers sync.WaitGroup
 
-	disableNmea  bool
-	isClosed     bool
-	err          movementsensor.LastError
-	lastposition movementsensor.LastPosition
+	disableNmea        bool
+	err                movementsensor.LastError
+	lastposition       movementsensor.LastPosition
+	lastcompassheading movementsensor.LastCompassHeading
```

Comment:
Same suggestion here about capitalizing the start of words in `lastPosition` and `lastCompassHeading`.